docs/Software Architecture Document/Components View/index.md

```diff
@@ -148,6 +148,21 @@ The view is represented by module and subsystem diagrams that show the system's
     - Not work for Android apps.
     - Not work for apps connected via SDL protocol version lower than 4.
 
+#### Plugin Manager
+  - *Responsibility*
+    - Loads all .so files from specific directory, checking if they’re exporting required methods
```

Comment:
@pestOO We do not support  Windows in release [4..4.0](https://github.com/orgs/smartdevicelink/projects/3)